### PR TITLE
Make home cards fully clickable

### PR DESCRIPTION
**Pull request by jacoobanderson**
## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Other (please describe):

## Issue

_Link to the issue (if available)_
#173 

_Please describe the current behavior._
Only a small part of the cards on the home page are clickable.

## Description of change & new behavior

_Please describe the changes you made and their new behavior._
The cards are now fully clickable, wrapped the whole card in anchor tags and adjusted the bootstrap classes accordingly.

## Screenshots

_For frontend updates, please include a screenshot._

## Technical Spec/Implementation Notes

Please check if your PR fulfills the following requirements:

- [ ] If it's a backend change, tests for the changes have been added and run successfully.
- [x] If it's a frontend change, Prettier has been run & all tests pass

### CODE REVIEW

**Comment by iporollo** [comment]
Hey @jacoobanderson , thanks for your help here! I changed the base branch to monosi:develop, could you rebase your branch with the latest monosi:develop code?

File: src/ui/src/pages/app/dashboard/Index/index.tsx [hunk 1]
```diff
@@ -22,45 +22,49 @@ const DashboardIndex: React.FC = () => {
...
-                        <h4>Monitor, Alert, Analyze</h4>
-                        <p className="card-text">Get insights on the quality of your data through automatic monitoring and alerting.</p>
+                      <div className="col-md-6 d-flex">
+                        <div className="card-body" style={{alignSelf: 'center'}}>
+                          <h4>Monitor, Alert, Analyze</h4>
+                          <p className="card-text">Get insights on the quality of your data through automatic monitoring and alerting.</p>
+                        </div>
                       </div>
-                    </div>
+                    </a>
```
Comment (iporollo) [inline src/ui/src/pages/app/dashboard/Index/index.tsx]: It seems like there's a missing </div> tag here. When I tried running it, I couldn't start the application. Were you able to run it locally to check?

**Review by iporollo — CHANGES_REQUESTED** [review]
When I pull this down I get an error that `<div>` isn't closed

**Comment by jacoobanderson** [comment]
Hi, the whole open source thing is new to me and I was unsure on how to run it locally, I did however check it in the browser but I obviously must have made a mistake while putting the code in. I will try to fix it.

**Comment by iporollo** [comment]
> Hi, the whole open source thing is new to me and I was unsure on how to run it locally, I did however check it in the browser but I obviously must have made a mistake while putting the code in. I will try to fix it.

No worries! Here's some docs on getting started with local development - https://docs.monosi.dev/docs/contributing/local-development . If you have docker, that will be the easiest way to run it (although the build times are long right now). Happy to help with any questions about the local setup / rebase / PR in our [Slack](https://monosi.dev/slack).

**Review by iporollo — APPROVED** [review]
Awesome!

**Pull request merged**